whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/acl/ResetPersonalObjectAccounting.java

```diff
@@ -17,8 +17,8 @@ public ResetPersonalObjectAccounting(final PersonalObjectAccounting personalObje
     }
 
     @Override
-    @Scheduled(cron = "0 0 0 * * *")
-    @SchedulerLock(name = "ResetPersonalObjectAccounting")
+    @Scheduled(cron = "@midnight")
+    @SchedulerLock(name = "ResetPersonalObjectAccounting", lockAtLeastFor = "23h")
```

Comment:
As suggested by @sbusk , this is now set to 23h (instead of 7h) to be explicit about the requirement to have these jobs only run once in space of 24hrs. He also suggested to have this in whois-internal...just to be really sure, even though we do not have the double job problem there...any thoughts @eshryane

Reply:
My only concern is whether this will prevent us from running those scheduled jobs during the day via JMX. Can you test to find out? If it's an issue we'll need a workaround (such as manually resetting the timestamp in the DB table beforehand).

Reply:
Done for whois-internal here https://github.com/RIPE-NCC/whois-internal/pull/434/commits/40ce8a6366ee9dbd612158d99408687e3bd3ec43

Reply:
> My only concern is whether this will prevent us from running those scheduled jobs during the day via JMX. Can you test to find out? If it's an issue we'll need a workaround (such as manually resetting the timestamp in the DB table beforehand).

It will definitely prevent you from running the job again during the day via JMX. The workaround is to update the lock time in the database but beware: shedlock also maintains a lock time cache in memory so you may also have to restart the node. Maybe we want to override the lock time in the runDailyScheduledTask JMX action so we can force a job to run when we need it. (it will currently silently refuse to run the job).

Reply:
I reduced this from 23h to 1h since the jobs should now run in parallel...I could remove it totally as it was before, but I think 1hr is safe enough and should not block being able to run jmx job manually

Reply:
I'd either leave it at 23h with a solid explanation as to why (to safeguard against a job running more than 1 time per day for whatever reason) or leave it out completely so it's the default (which is more intuitive).
Setting it to 1h doesn't prevent the job from running more than once per day and also differs from the default so it's confusing as to where the 1h comes from.

Reply:
I'll go with leaving it out. Having it as 23h essentially nullifies the ability to run the jobs via JMX.